src/cljs/org/broadinstitute/firecloud_ui/common.cljs

```diff
@@ -131,3 +131,23 @@
 
 (def root-entity-types
   ["participant" "sample" "pair" "participant_set" "sample_set" "pair_set"])
+
+(defn count-workflows [entity]
+  (case (entity "entityType")
+    "participant_set" (count (get-in entity ["attributes" "participants"]))
+    "sample_set" (count (get-in entity ["attributes" "samples"]))
+    "pair_set" (count (get-in entity ["attributes" "pairs"]))
+    1))
+
+(defn- make-pair [single-type]
+  [{:header (str "# " single-type) :starting-width 110}
+   #(count (get-in % ["attributes" single-type]))])
+
+(defn make-count-column
+  "When the given entity type is a *_set, returns [column_header fn-from-row-to-count]"
+  [entity-type]
+  (case entity-type
```

Comment:
I think you should break the "sample_set" -> "samples", "participant_set" -> "participants", "pair_set" -> "pairs" mappings out into a separate value/function somewhere, and query that map from both count-workflows and make-count-column (as well as from GAWB-701 when that gets implemented).